examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -152,6 +160,11 @@
   ],
   "parameters": [
     {
+      "name": "DEPLOYMENT_LIMIT_SIZE",
```

Comment:
In the mean time (while I sort out if this PR is still needed), I'll be:
1) changing the name to POD_MEMORY_LIMIT
2) updating the description, pulling some description around impact from the doc:  https://docs.openshift.com/enterprise/3.1/dev_guide/compute_resources.html#memory-limits
3) for "displayName", my thought was for each name to convert capital letters to lower case were appropriate, and underscores to spaces

I'll give a heads up when I push.